tools/network/backup/backup.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+import argparse
+import logging
+import os
+
+from brigade.core import InitBrigade
+from brigade.plugins.functions.text import print_result
+from brigade.plugins.tasks.files import write_file
+from brigade.plugins.tasks.networking import napalm_get
+
+
+def backup(task, path):
+    r = task.run(
+        task=napalm_get,
+        getters=["config"],
+        severity=logging.DEBUG,
+    )
+    task.run(
+        task=write_file,
+        filename=f"{path}/{task.host}",
```

Comment:
If we do this perhaps we should state in the README.md that the examples are written for Python 3.6?

Reply:
Adding a note